bootstrap/http_server.go

```diff
@@ -31,13 +36,18 @@ func createFiberApp() *fiber.App {
 	return fiber.New()
 }
 
-func createAPIRoutes(cities *controllers.CityController, forecasts *controllers.ForecastController) []controllers.Route {
+func createAPIRoutes(cities *controllers.CityController, forecasts *controllers.ForecastController, weather *services.WeatherApiService) []controllers.Route {
 	return slices.Concat(
 		cities.GetRoutes(),
 		forecasts.GetRoutes(),
+		weather.GetRoutes(),
 	)
 }
 
+func createWeatherApiService(config *config.ApplicationConfiguration) *services.WeatherApiService {
+	client := &http.Client{}
```

Comment:
It'll be great to extract the HTTP Client as a separate "bean" so that you can be able to configure it etc. without introducing too much clutter in here.